src/components/users/register/register.js

```diff
@@ -52,14 +75,29 @@ export default class Register extends React.Component {
     }
 
     updateStorageLocation = (storageLocationUpdate) => {
```

Comment:
This may not be reusable for generic user address, since it will only update storageLocationUpdate.